types/express-jwt/index.d.ts

```diff
@@ -62,3 +62,14 @@ declare namespace jwt {
         constructor(code: ErrorCode, error: { message: string });
     }
 }
+
+declare global {
+    namespace Express {
+        // tslint:disable-next-line:no-empty-interface
+        interface User {}
+
+        interface Request {
+            user: User;
```

Comment:
```suggestion
            user?: User;
```
can this be optional, please? like:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/passport/index.d.ts#L24
Not every request/response bears session info of some kind, thx!